modules/FvwmScript/FvwmScript.c

```diff
@@ -281,6 +283,11 @@ void ParseOptions(void)
 	CopyStringWithQuotes(&x11base->font,&tline[22]);
       else if (strncasecmp(tline,"Colorset",8) == 0)
 	LoadColorset(&tline[8]);
+      else if (strncasecmp(tline,"*FvwmScriptDefaultCursor",24) == 0)
```

Comment:
Do you really need the leading `*` here?

Also, can we not just call it `DefaultCursor`?

Reply:
> Hi @NsCDE,

Hi Thomas,

Sorry for a delay. It went to sleep. :-)

I will answer about both issues you highlighted here:

> Yep -- all looks good. Before I even looked at the change, the diffstat looked quite large for configuring cursors, but then I can see there's duplication for each widget.

It was like that before, and wanting to be non-intrusive, I have simply armored this hard coding of cursor with my new condition in all that places.

> One tiny query, otherwise the rest looks good.
> Do you really need the leading * here?
> Also, can we not just call it DefaultCursor?

This part of the configuration parsing code is not really clear to me. If you look above line 286 in FvwmScript.c you will see that I have simply mimicked the way other global FvwmScript options are defined. Looks like this "*FvwmScript<bla>" part of the string is someware somehow split into "*FvwmScript: " and "DefaultCursor" afterwards, because this is exactly what I have put in configuration while testing this:

`*FvwmScript: DefaultCursor left_ptr`

P. S.
Soon, when I catch some time there there will be TextField colorset unwanted inheritance of foreground color bug report in FvwmScript. I will try to figure it out and fix myself, but it doesn't look really readable, so maybe it will be an issue. This is bug also in 2.X.